### ISSUE TITLE
german umlaut not correctly fixed

### ISSUE THREAD

**Issue by forty875** [rspeer/python-ftfy#188]
**German umlaut example:**
```
print(ftfy.fix_text('ÃŸ'))
print(ftfy.fix_text('Ãœ'))
print(ftfy.fix_text('Ã¼'))
print(ftfy.fix_text('Ã–'))
print(ftfy.fix_text('Ã¶'))
print(ftfy.fix_text('Ã„'))
print(ftfy.fix_text('Ã¤'))
print(ftfy.fix_text('RUF MICH ZURÃœCK.'))
```
**Output:**
```
ß
Ãœ
ü
Ö
ö
Ä
ä
RUF MICH ZURÃœCK.
```
The german umlaut upper Ü  Ãœ is not correctly fixed...

**Comment by rspeer** [comment]
Thanks, I can expand the heuristic to account for this case.

**Comment by rspeer** [comment]
Released in 6.3.
